core/backend/src/ECSqlStatement.ts

```diff
@@ -332,46 +336,43 @@ export class ECSqlStatement implements IterableIterator<any>, IDisposable {
    * - [Code Samples]($docs/learning/backend/ECSQLCodeExamples#working-with-the-query-result)
    */
   public getRow(args?: ECSqlRowArg): any {
-    args = args ?? {};
+    if (!this._stmt)
+      throw new Error("ECSqlStatement is not prepared");
 
-    // Set default rowFormat if not provided
+    args = args ?? {};
     if (args.rowFormat === undefined) {
       args.rowFormat = QueryRowFormat.UseJsPropertyNames;
     }
-
-    const toRowOptions = {
-      classIdsToClassNames: true,
-      rowFormat: args.rowFormat,
-    };
-    const resp = this._stmt!.toRow(toRowOptions); // eslint-disable-line @typescript-eslint/no-non-null-assertion
+    if (args.classIdsToClassNames === undefined) {
+      args.classIdsToClassNames = true;
```

Comment:
Can you explain why its default true?

Reply:
I am removing it. It's previously set to true and is noop when useJsNames is true. It's useful when not using jsName.

Reply:
Should your response or some variation of it be documented on the classIdsToClassNames property?

Reply:
Before your PR wasn't `classIdsToClassNames` always true? Why shouldn't it stay that way for 4.x? If someone currently has a query that specifies a rowFormat which is not `useJsPropertyNames`, and they update to a 4.x version containing this PR, wouldn't the output of their query change?  

I'm not talking about the other one `donotconvertclassidstoclassnameswhenaliased` which I assume is the one that was added for backward compatibility, since the comment above it says 'it is only here to maintain compatibility with 4.x'

Reply:
Yes that is correct, though this has been rolled out just recently and that rollout is what broke hatch. It's not advertised (and we have no choice to make it public given its input to a public api otherwise we would have marked it beta). I will not worry about it.

Reply:
Ah okay, I see now thanks. Didn't realize that the `toRowOptions` were part of the PR that broke hatch.